### PR TITLE
Feature/hashes plugin

### PR DESCRIPTION
**Pull request by mabruras**

### CODE REVIEW

File: hashes/__init__.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        sha256.update(chunk)
+
+    return dict({
+        'md5': md5.hexdigest(),
+        'sha1': sha1.hexdigest(),
+        'sha256': sha256.hexdigest(),
+    })
+
+
+def _chunk_file_read(data, chunk_size=1024):
```
Comment (mabruras) [inline hashes/__init__.py:41]: Add custom configuration for `chunk_size`

File: hashes/scripts/init.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS hashes (
+  id             TEXT NOT NULL PRIMARY KEY,
+  uuid           TEXT NOT NULL,
```
Comment (gullskatten) [inline hashes/scripts/init.sql:3]: Create indexes for `uuid`, `md5`, `sha1` and `sha256`?

File: hashes/scripts/init.sql [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+  meta_location  TEXT,
+  data_location  TEXT,
+  md5            TEXT,
+  sha1           TEXT,
+  sha256         TEXT,
+  created_at     TIMESTAMPTZ DEFAULT Now()
+);
```
Comment (gullskatten) [inline hashes/scripts/init.sql:10]:
```suggestion
);

CREATE INDEX IF NOT EXISTS hashes_uuid_idx ON hashes (uuid);
CREATE INDEX IF NOT EXISTS hashes_md5_idx ON hashes (md5);
CREATE INDEX IF NOT EXISTS hashes_sha1_idx ON hashes (sha1);
CREATE INDEX IF NOT EXISTS hashes_sha256_idx ON hashes (sha256);
```

File: hashes/scripts/init.sql (hunk 2 above)
Comment (gullskatten) [reply hashes/scripts/init.sql:3]: Change suggested below.

**Review by gullskatten — APPROVED** [review]
Looks ok!

**Pull request merged**